src/current/cockroachcloud/managed-backups.md

```diff
@@ -1,23 +1,109 @@
+
+Once a cluster is deleted, Cockroach Labs retains the backup for 30 days. The retained backups are not available for restore using the Cloud Console. If an organization is deleted, you will lose access to all of the managed backups that Cockroach Labs has taken of the cluster after 30 days.
+
+To restore a backup from a deleted cluster, you must contact the [Cockroach Labs Support team](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/support-resources).
+
+### Standard and Advanced clusters
+
+In {{ site.data.products.standard }} and {{ site.data.products.advanced }} clusters you can configure the [frequency](#frequency) and [retention](#retention) of managed backups. For details on setting these options with the Cloud API, refer to the [Use the CockroachDB Cloud API](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/cloud-api) page. You can view your configured managed backup settins in the [Cloud Console]({% link cockroachcloud/cluster-overview-page.md %}).
+
+#### Frequency
+
+You can configure how frequently Cockroach Labs takes backups, which will determine the cluster's [RPO]({% link {{site.current_cloud_version}}/disaster-recovery-overview.md %}#resilience-strategy).
+
+You can set backup frequency to one of the following options:
+
+- Disabled
+- 5 minutes
+- 10 minutes
+- 30 minutes
+- 1 hour (default)
+- 4 hours
+- 24 hours
+
+#### Retention
+
+You can set your retention duration **once**. The available retention options are:
+
+- 2 days
+- 7 days
+- 30 days (default)
+- 90 days
+- 365 days
+
+If you need to adjust the retention setting after your initial selection, contact the [Cockroach Labs Support team](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/support-resources).
+
+{% include cockroachcloud/backups/retention-deleted-cluster.md %}
+
+### Upgrades and downgrades
 
-{% include cockroachcloud/backups/managed-service-backups-frequency.md %}
+If you have upgraded from a {{ site.data.products.basic }} cluster to a {{ site.data.products.standard }} cluster, the existing backup schedules will still apply, but you can then configure the frequency and retention of future managed backups in the {{ site.data.products.standard }} cluster.
 
-To access your managed-service backups, select a cluster from the [**Clusters** page]({% link cockroachcloud/cluster-management.md %}#view-clusters-page), then click **Backup and Restore** in the **Data** section of the left-side navigation.
+If you have downgraded from a {{ site.data.products.standard }} cluster to a {{ site.data.products.basic }} cluster, existing managed backups will be retained for the configured retention duration. The default managed backups in {{ site.data.products.basic }} clusters will be taken every 24 hours and have a 30-day retention.
+
+
```

Comment:
All content from here down is old, still to do.

Reply:
thanks! just reviewed the first bit.